src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -113,8 +113,13 @@ internal static bool DoSslHandshake(SafeSslHandle context, byte[] recvBuf, int r
 
         }
 
-        internal static int Encrypt(SafeSslHandle context, byte[] buffer, int offset, int count, int bufferCapacity, out libssl.SslErrorCode errorCode)
+        internal static int Encrypt(SafeSslHandle context, byte[] buffer, int offset, int count, out libssl.SslErrorCode errorCode)
         {
+            Debug.Assert(buffer != null);
```

Comment:
Nit: I vaguely remember @stephentoub mentioning some guidance about accompanying asserts with message strings since there is some issue with line numbers displayed by the assert logic